llvm/lib/CodeGen/SelectionDAG/LegalizeFloatTypes.cpp

```diff
@@ -60,7 +60,9 @@ void DAGTypeLegalizer::SoftenFloatResult(SDNode *N, unsigned ResNo) {
 #endif
     report_fatal_error("Do not know how to soften the result of this "
                        "operator!");
-
+    case ISD::EXTRACT_ELEMENT:
+      R = SoftenFloatRes_EXTRACT_ELEMENT(N);
```

Comment:
put these on a single line like the rest of the switch (I've committed a9d963fdf81dc5c6221a0e915821877c35096aff to hopefully stop clang-format warning about this).

Reply:
```cpp
    case ISD::EXTRACT_ELEMENT: R = SoftenFloatRes_EXTRACT_ELEMENT(N); break;
    case ISD::ARITH_FENCE: R = SoftenFloatRes_ARITH_FENCE(N); break;
    case ISD::EXTRACT_VECTOR_ELT:
      R = SoftenFloatRes_EXTRACT_VECTOR_ELT(N, ResNo); break;
```

or

```cpp
    case ISD::EXTRACT_ELEMENT:
      R = SoftenFloatRes_EXTRACT_ELEMENT(N); break;
    case ISD::ARITH_FENCE: R = SoftenFloatRes_ARITH_FENCE(N); break;
    case ISD::EXTRACT_VECTOR_ELT:
      R = SoftenFloatRes_EXTRACT_VECTOR_ELT(N, ResNo); break;
```

since `R` cannot be aligned...

Reply:
IIRC SoftenFloatRes_EXTRACT_VECTOR_ELT has always been too long - alignment isn't very important tbh, its just about reducing loc as long as its still col80